core/src/main/java/org/ehcache/core/Ehcache.java

```diff
@@ -326,24 +327,10 @@ public void clear() {
       return Collections.emptyMap();
     }
 
-    Function<Iterable<? extends K>, Iterable<? extends Map.Entry<? extends K, ? extends V>>> computeFunction =
-        new Function<Iterable<? extends K>, Iterable<? extends Map.Entry<? extends K, ? extends V>>>() {
-      @Override
-      public Iterable<? extends Map.Entry<? extends K, ? extends V>> apply(Iterable<? extends K> keys) {
-        Map<K, V> computeResult = new LinkedHashMap<K ,V>();
-
-        // put all the entries to get ordering correct
-        for (K key : keys) {
-          computeResult.put(key, null);
-        }
-
-        return computeResult.entrySet();
-      }
-    };
-
+    GetAllFunction<K, V> getAllFunction = new GetAllFunction<K, V>();
```

Comment:
I wouldn't bother with a variable, just pass the `new GetAllFunction<K, V>()` to the `bulkComputeIfAbsent`. That would make the code an tad cleaner.